docs/low-code-sql/getting-started-with-low-code-sql.md

```diff
@@ -0,0 +1,382 @@
+---
```

Comment:
@atbida Could we move this under the `Getting Started` section?

Reply:
We can have two sections there - `Spark with Databricks` & `SQL with Databricks`